kyuubi-server/src/main/scala/org/apache/kyuubi/operation/BatchJobSubmission.scala

```diff
@@ -300,6 +308,20 @@ class BatchJobSubmission(
     }
   }
 
+  private def deleteTempLocalResourceFile(filePath: String): Unit = {
+    try {
+      val uri = new URI(filePath)
+      uri.getScheme match {
```

Comment:
seems it can be only local file, because the uploaded resource is internal and full control.

Reply:
Agree. And changed to use `FileUtils.deleteQuietly` directly, with no exception need to be catched.